lib/rspec/mocks/mutate_const.rb

```diff
@@ -105,6 +105,10 @@ class ConstantMutator
       #  so you can stub constants in other contexts (e.g. helper
       #  classes).
       def self.stub(constant_name, value, options={})
+        raise ArgumentError,
+              "Please provide a string as constant name" \
+              " You provided a #{constant_name.class.name}" unless constant_name.is_a?(String)
```

Comment:
Also, the formatting on this can be improved, I think:

```
Please provide a string as constant name You provided a SomeClass
```

Instead, how about:

```
`stub_const` requires a String, but you provided a SomeClass
```